client/src/app/tasks/components/edit-dialog/edit-dialog.component.spec.ts

```diff
@@ -8,9 +11,16 @@ describe('EditDialogComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ EditDialogComponent ]
-    })
-    .compileComponents();
+      imports: [ReactiveFormsModule, MatDialogModule],
+      declarations: [EditDialogComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      providers: [
+        {
+          provide: MAT_DIALOG_DATA,
+          useValue: { title: '', completed: false, importance: 0 },
```

Comment:
Lest add in in separate file. As I can see in task-item component you use the same object